### PR TITLE
Add examples for Title and BossBar

### PR DESCRIPTION
**Pull request by PrimordialMoros**
Adds examples for constructing Titles and Boss Bars.
More examples and information to be added soon.

### CODE REVIEW

**Comment by kashike** [comment]
Thanks for the pull request, we appreciate it! We will merge when you tell us it is ready.

File: source/bossbar.rst [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+=========
+
+Constructing a Boss Bar
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Boss Bars are composed of:
+* A component which represents the text shown
+* A percentage, how full the bar should be from 0 (empty) to 1 (full)
+* A color, the main color of the boss bar
+* A style/overlay graphic that allows you to select the amount notches on the boss bar
```
Comment (MiniDigger) [inline source/bossbar.rst:12]:
```suggestion
* A style/overlay graphic that allows you to select the amount of notches on the boss bar
```

**Comment by PrimordialMoros** [comment]
Lists require the indentation to work, not sure why the checker complains

**Review by kashike — APPROVED** [review]

**Comment by kashike** [comment]
Thank you!

**Pull request merged**